packages/kbn-pm/dist/index.js

```diff
@@ -47554,7 +47554,7 @@ __webpack_require__.r(__webpack_exports__);
 /* harmony import */ var _scripts__WEBPACK_IMPORTED_MODULE_7__ = __webpack_require__(562);
```

Comment:
isn't it interesting the `dist/` dir is pushed to GitHub?

Reply:
kbn/pm is used to install dependencies with `kbn bootstrap` before any node module is installed, so it has to bundle the node_modules it needs in a file that git can put in place, which is why it's committed to the repo.